streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableKTableJoinMerger.java

```diff
@@ -119,7 +119,7 @@ public void init(final ProcessorContext<K, Change<V>> context) {
         public void process(final Record<K, Change<V>> record) {
             if (queryableName != null) {
                 store.put(record.key(), ValueAndTimestamp.make(record.value().newValue, record.timestamp()));
-                tupleForwarder.maybeForward(record.key(), record.value().newValue, sendOldValues ? record.value().oldValue : null);
+                tupleForwarder.maybeForward(record.withValue(new Change<>(record.value().newValue, sendOldValues ? record.value().oldValue : null)));
```

Comment:
Huh. I bet it would be ok to just:
```suggestion
                tupleForwarder.maybeForward(record);
```

Reply:
😅